agent/src/Factories/BrowserFactory.php

```diff
@@ -37,7 +36,7 @@ public function __invoke(
         }
 
         return new NightwatchBrowser($browser, [
-            'user-agent' => fn () => 'NightwatchAgent/'.$this->packageVersion->get(),
+            'user-agent' => fn () => 'NightwatchAgent/'.$this->packageVersion,
```

Comment:
We should remove the lazy header concept entirely and this should be passed through to the invoke `__invoke` method like all the other static headers.